### ISSUE TITLE
Tag Format Issue

### ISSUE THREAD

**Issue by acherrera** [jonathan-s/pelican-obsidian#5]
# Tags Messing Up During Formatting

I have frontmatter in my markdown files that looks like the following. The format of the tags here works as expected:
```
---
Title: Some  Recipe
Date: 2023-11-13
Slug: some_recipe
tags: food, recipe, vegetarian, bread
Author: acherrera
Summary: How to make something
---
```

However, when the tags are formatted as the following, it does not work as expected:

```
tags
 - food
 - recipe
 - vegetarian
 - bread
```

Instead, everything from tags down is rendered as Markdown instead of being put in the metadata. I thought it was due to the metadata tag converter, but it appears to be in the link formatter. Removing all wikilinks from the post causes the tags to be processed as expected.

**Comment by arulrajnet** [comment]
@acherrera I have a simple [bash script ](https://github.com/arulrajnet/blog.arulraj.net/blob/master/convert-tags-obsidian-markdown.sh)to fix this. Then tie that with [Makefile](https://github.com/arulrajnet/blog.arulraj.net/blob/master/Makefile#L98).

**Comment by jonathan-s** [comment]
@arulrajnet and @acherrera Fixed now.

**Comment by arulrajnet** [comment]
I have taken the fix. After this the first letter of a tag is missing in all of my posts.

![2024-12-26_21-09_1](https://github.com/user-attachments/assets/2aa27f4e-fc85-4c49-b5d0-f6b96bbb6ada)

![2024-12-26_21-09](https://github.com/user-attachments/assets/f5708b3d-6c41-4a1a-9659-395a4b484d21)


https://www.arulraj.net/2024/12/no-more-basic-auth-htpasswd-with-oauth2-proxy#show-disqus

https://www.arulraj.net/2024/12/pi-hole-as-internet-parental-control#show-disqus

**Comment by arulrajnet** [comment]
@jonathan-s Fixed in this #10 PR
